Firestore/core/src/remote/serializer.cc

```diff
@@ -637,6 +637,11 @@ google_firestore_v1_Target Serializer::EncodeTarget(
         nanopb::CopyBytesArray(target_data.resume_token().get());
   }
 
+  if (target_data.expected_count()) {
```

Comment:
Here, the logic on resumeToken is different from web and Android, do we need to make them consistent?

Reply:
add .has_value() to condition.